google-dracut-config.spec

```diff
@@ -25,6 +25,8 @@ Group:          System/Management
 URL:            https://github.com/SUSE-Enceladus/google-dracut-config
 Source:         %{name}-%{version}.tar.gz
 BuildRequires:  dracut
+BuildRequires:  sed
+BuildRequires:  find
```

Comment:
sed and find are required for installation not building, so this should be `Requires` instead of `BuildRequires`. Also, the package that contains `find` is `findutils`, although I think `find` works too.